scripts/validate.py

```diff
@@ -194,17 +195,27 @@ def validate_experiment(item):
 
 def validate(schema_name, src_path):
     schema = load_schema(schema_name)
+    new_experiments_schema = load_schema("messaging-experiments-82")
     check_action = schema_name in ['cfr']
 
     with open(src_path, "r") as f:
         items = json.loads(f.read())
         for item in items:
             try:
                 print("Validate schema for {}".format(item["id"]))
-                jsonschema.validate(instance=item, schema=schema)
-                # If it's an experiment we want to evaluate the branches
                 if "arguments" in item:
+                    try:
+                        print("Validate {} with Experiments schema 82+".format(item["id"]))
+                        jsonschema.validate(instance=item, schema=new_experiments_schema)
+                    except ValidationError as err:
+                        match = best_match([err])
+                        print("Validation error against the new experiments schema: {}"
+                              .format(match.message))
```

Comment:
Lots of `jsonschema.validate()` calls in this block 😁. As a short-term mitigation, I think this is fine. Could you add one more print here to signify that it's going to try with the old schema if the new one didn't pass?